src/modules/SourceDialogs.rb

```diff
@@ -44,7 +44,7 @@ class SourceDialogsClass < Module
       # radio button
       :samba             => N_("&SMB/CIFS"),
       # radio button
-      :nfs               => N_("&NFS..."),
+      :nfs               => N_("NF&S..."),
```

Comment:
ncurses with Y2DEBUG revealed me its logic: it first resolved the conflicts among the radio buttons. Then it assigned the Next button label. Previously, NFS kept its *preferred* key N, so Next lost against it. Now, even though the radio buttons have a conflict, nobody gets N so Next can keep it.

The shortcuts are still different in Qt and ncurses, only Next now gets N.

I think it is acceptable. LGTM.